### DISCUSSION TITLE
🤔 How can we use Instagram Order feature(order button)

### DISCUSSION THREAD

**Discussion by mohsenxad** [lettucesocial/com-lettucesocial-customer-panel#75] [Ideas]
### 🤔 How can we use Instagram Order feature(order button)

There is a button down bellow in Instagram DM.
you can create an order under that
I don't know the payment flow in Instagram
can we test it?

**Comment by maxzad** [comment]
@mohsenxad 
I am not sure how helpful would be to process payments through IG.
In general, IG is not a place where people would feel comfortable making payments through/entering their credit card numbers, especially in DMs. DMs are most associated with scams/etc.
Currently, our payment processing flow has not been a problem, and I have not heard any negative feedback from any customers. In contrast, many customers are very familiar with the flow and view since some of them also use Stripe for their online sales.

We can come back to this in the future if there are any ideas that require us to keep the user inside IG and not take them outside of the IG app for processing payments and then bring them back where they were in IG.
